brands/amenity/cafe.json

```diff
@@ -1062,6 +1062,29 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Tchibo": {
+    "locationSet": {"include": [
+      "at",
+      "cz",
+      "hu",
+      "is",
+      "nl",
```

Comment:
According to the [Dutch Wikipedia article](https://nl.wikipedia.org/wiki/Tchibo#Geschiedenis), the last location in the Netherlands closed in 2007.

Reply:
Updated.